pages/care.html

```diff
@@ -0,0 +1,144 @@
+            <a class="menu-link" href="#">Care</a>
+          </li>
+          <li>
+            <a class="menu-link" href="facts.html">Facts</a>
+          </li>
+          <li>
+            <a class="menu-link" href="gallery.html">Gallery</a>
+          </li>
+        </ul>
+      </nav>
+  </header>
+  <main>
+    <div class="intro">
+      <div id="summary">
+          <p> This is a quick guide on how to take care of your plants!</p>
+      </div>
+      <div id="table-of-contents">
+        <h2>Plant Care</h2>
+        <ol>
+          <li><a href="#Light">Light 🌞</a></li>
+          <li><a href="#Watering">Watering 💦</a></li>
+          <li><a href="#Soil">Soil and Potting 🪴</a></li>
+          <li><a href="#Temperature">Temperature and Humidity 🌡</a></li>
+        </ol>
+        <h3>Other tips and tricks</h3>
+        <ul>
+          <li><a href="#Troubleshooting">Troubleshooting common issues</a></li>
+          <li><a href="#Green_Thumb">Taking your Green Thumb up a notch!</a></li>
+        </ul>
+      </div>
+    </div>
+    
+    <div class="content">
+
+      <div id="Light">
+        <div class="text">
+          <h3>Light 🌞</h3>
+          <p> Different plants thrive under different amounts of light so it's important to know how much your specific plant needs. Most indoor plants prefer indirect light.</p>
+          <ul>
+            <li><strong>Shade/Low light:</strong> Very little natural light; far from light sources</li>
+            <li><strong>Low Indirect Light:</strong>  Light that's filtered or reflected; a dim room with a north-facing window or a spot far from a window</li>
+            <li><strong>Bright Indirect Light aka "Medium Light":</strong> A bright light source, but not in the direct path of the sun. Most common for house plants!</li>
+            <li><strong>Partial Sun:</strong>  Direct sun, but not intense "peak" sun heat/light</li>
+            <li><strong>Full Sun:</strong> Direct, unfiltered sun for several hours a day.
+            </li>
+          </ul>
+        </div>
+        <img src="/assets/lighting_guide.jpg" alt="lighting guide" width="400" height="350">
+        
+      </div>
+      
+      
+      <div id="Watering">
+        <div class="text">
+          <h3>Watering💦</h3>
+          <p>
+            Identify your specific plant's watering needs. A rule of (green) thumb is that if the top inch of soil is dry, it's time to water!
+            Make sure your plants have proper drainage holes or else standing water can lead to root rot
+          </p>
+        </div>
+        <img src="/assets/watering_chart.jpg" alt="watering guide" height="500" width="350">
+      </div>
+
+      <div id="Soil">
+        <div class="text">
+          <h3>Soil and Potting</h3>
+          <p>
+            <strong>Use the Right Soil:</strong> Most houseplants do well in a general-purpose potting mix. However, some plants like succulents or cacti need a fast-draining soil mix to prevent root rot.
+            <br>
```

Comment:
This comment applies to all the other line break elements in the project:

We want to be pretty sparing with our use of the `br` tag and try to save it for places where a line break is really meaningful to the content (Examples from the MDN docs on `br` are poems and postal addresses). It's considered better form to use CSS to set margins or create space around an element.

A primary reason to avoid `br`s is that they can make navigation with screen readers more difficult. I recommend taking a look at the accessibility concerns section of the MDN's `br` reference. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns